src/utils/fs.spec.js

```diff
@@ -83,14 +83,15 @@ describe('findUpwardsFile()', () => {
   })
 
   it('looks for file in the supplied directory', async () => {
+    const path = jest.requireActual('path')
```

Comment:
Do we need `jest.requireActual('path')`? I don't think we are mocking path anywhere.
Just `const path = require('path')` at the top of the file should be enough I think, since we are using path in multiple tests.